tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -209,8 +209,10 @@ export function makeMatMulPackedVec4Source(
     let batch = ${splitK ? '0' : 'i32(globalId.z)'};
     let globalRowStart = i32(workgroupId.y) * ${tileAOuter};
 
-    let numTiles = ${splitK ? '1' : '(uniforms.dimInner - 1) / TileInner + 1'};
-    var kStart = ${splitK ? 'i32(globalId.z) * TileInner' : '0'};
+    let numTiles = ${
+      splitK ? `${splitedDimInner / tileInner}` :
```

Comment:
Nit. Do we need to change it to `(splitedDimInner - 1) / TileInner + 1` if `splitedDimInner % tileInner` is not equal to 0